assets/css/fonts.css

```diff
@@ -40,6 +40,20 @@
     src: url('/fonts/ExpensifyMono-Bold.woff2') format('woff2'),  url('/fonts/ExpensifyMono-Bold.woff') format('woff');
 }
 
+@font-face {
```

Comment:
@shawnborton I still need to add screenshots for this PR, but can you please do an early review of these css rules?

Reply:
I think the right way to do this would be to use the genera font family name, and the weight to be something like 500. 

So something like:

```
font-family: "Expensify New Kansas";
font-weight: 500,
font-style: normal;
src: url('/fonts/ExpensifyNewKansas-Medium.otf') format('otf');
```

Then when we go to create styles for our headlines, we always need to make sure we use `font-weight: 500` for the weight, and then `font-family: "Expensify New Kansas"` for the weight

Reply:
This is outdated and this PR no longer includes any css changes